.dockerignore

```diff
@@ -0,0 +1,18 @@
+# Documentation, might leak version number
```

Comment:
Actually the version number is leaked in PrivateBin itself (e.g. on the bottom of the site it is shown and when loading some JS files it is appended).